src/main/java/fr/polytech/sim/log/NamedLogger.java

```diff
@@ -16,4 +16,14 @@ protected NamedLogger(String name) {
         this.name = name;
     }
 
+    @Override
+    public void log(String format, Object... args) {
+        String entry = String.format(format, args);
+        logMethod(String.format("%s\t%s\n", this.name, entry));
+    }
+
+    public abstract void logMethod(String message);
```

Comment:
Cette méthode devrai être `protected`.
```suggestion
    protected abstract void logMethod(String message);
```